### PR TITLE
ellipsis part 2

### PR DESCRIPTION
**Pull request by brianteeman**
oops missed updating these two files to use the html entity instead of the three dots

see #38638

### CODE REVIEW

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 114db67cf738187e54b73142e8e17a658d59e56c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38710">issues.joomla.org/tracker/joomla-cms/38710</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 114db67cf738187e54b73142e8e17a658d59e56c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38710">issues.joomla.org/tracker/joomla-cms/38710</a>.</sub>

**Comment by Abernyte-Git** [comment]
I have tested this item :white_check_mark: successfully on 114db67cf738187e54b73142e8e17a658d59e56c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38710">issues.joomla.org/tracker/joomla-cms/38710</a>.</sub>

**Comment by HLeithner** [comment]
as commented in the other pr can we please use the proper utf8 character and not the html encoded string.

**Comment by brianteeman** [comment]
No I will not change it and we have plenty of html in our strings.

&hellip is easier to type and visually an editor will not mistakenly type ... when they mean …

As already explained we must use an ellipsis for accessibility and changing to use the utf8 … from the ... will not help as translators etc will not SEE that they are different and will continue to type ... especially as its much easier to type ... than it is to look up the utf8 character. It's certainly easier for me to type &hellip than it is to open a character map and locate the utf8 map and then locate the ellipsis and insert it into a document

**Comment by Razzo1987** [comment]
I have tested this item :white_check_mark: successfully on 114db67cf738187e54b73142e8e17a658d59e56c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38710">issues.joomla.org/tracker/joomla-cms/38710</a>.</sub>

**Comment by infograf768** [comment]
The problem using the web ellipsis character (UTF8 or hellip;) instead of 3 periods is the space between the dots. It entirely depends on the font used. In some cases the ellipsis is hardly readable.

Example:

Screenshot 2022-09-07 at 10 50 59

**Comment by brianteeman** [comment]
using three individual dots is simply wrong

**Comment by Razzo1987** [comment]
3 dot, utf8 or ellipsis?

How does a screen reader behave for blind users?
I don't know the answer, but I think it might help us figure out what not to use.

**Comment by brianteeman** [comment]
> I don't know the answer, but I think it might help us figure out what not to use.

I already did the research ;)

Sadly there is no uniform answer for all screen readers and it also depends what mode they are in.

https://bighack.org/accessibility-screen-readers-special-characters-and-unicode-symbols/

https://tosbourn.com/the-importance-of-using-the-ellipsis-character/

**Comment by brianteeman** [comment]
sorry JM the english is fine

**Comment by brianteeman** [comment]
no one is forced to merge this

**Comment by N6REJ** [comment]
@brianteeman stop being defensive.  JM brings up a valid point.

**Comment by N6REJ** [comment]
@chmst can the accessibility team give some feedback?

**Comment by brianteeman** [comment]
@N6REJ please educate me with real world examples of the strings chanhged in this PR how it is ever wrong in english and please provide documented evidence of that

**Comment by N6REJ** [comment]
he's not talking about the string he's talking about the string behavior and THAT as he showed is currently wrong which was the OP point

**Comment by brianteeman** [comment]
Never mind. Ignore what I said. Merge or reject - not my call

File: language/en-GB/com_media.ini [hunk 1]
```diff
@@ -98,7 +98,7 @@ COM_MEDIA_PLEASE_SELECT_ITEM="Please select item."
 COM_MEDIA_RENAME="Rename"
 COM_MEDIA_RENAME_ERROR="Error renaming item."
 COM_MEDIA_RENAME_SUCCESS="Item renamed."
-COM_MEDIA_SEARCH="Search..."
+COM_MEDIA_SEARCH="Search &hellip;"
```
Comment (Kostelano) [inline language/en-GB/com_media.ini:101]: You need to remove the ellipsis (in the admin panel without the ellipsis), as this creates a problem.

Screenshot_1

**Pull request closed without merging**